### ISSUE TITLE
-library and -module-load-dir flag don't do anything

### ISSUE THREAD

**Issue by awstlaur** [brownplt/pyret-lang#958]
Looking at `pyret.arr`, it seems like these flags don't do anything, except for `-library` incidentally implying `-no-check-mode`.

I assume they are remnants of the old module system.

**Comment by jpolitz** [comment]
Yup, they can be removed.  `-library` is handled by the builtin locator implementation skipping some imports (so `lists` doesn't get imported into `lists.arr` and cause issues, for example), and `--module-load-dir` is replaced by `--builtin-js-dir` and `--builtin-arr-dir`.
